_pages/install/integrations/opf-integration/opf-child-pages/2-quickbuy.md

```diff
@@ -0,0 +1,119 @@
+
+For more information about the available types of Apple Pay buttons, see [Displaying Apple Pay Buttons Using CSS](https://developer.apple.com/documentation/apple_pay_on_the_web/displaying_apple_pay_buttons_using_css) in the official Apple developer documentation.
+
+### Modifying Credit Card Parameters for Apple Pay
+
+The card parameters configuration for Apple Pay is hard-coded in the `ApplePayService` class in `apple-pay.service.ts`. You can modify the configuration by extending `ApplePayService` and overwriting the following object:
+
+```ts
+  protected readonly defaultApplePayCardParameters: any = {
+    shippingMethods: [],
+    merchantCapabilities: ['supports3DS'],
+    supportedNetworks: ['visa', 'masterCard', 'amex', 'discover'],
+    requiredShippingContactFields: ['email', 'name', 'postalAddress'],
+    requiredBillingContactFields: ['email', 'name', 'postalAddress'],
+  };
+```
+
+For more information on extending services in Spartacus, see https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/aa417173fe4a4ba5a473c93eb730a417/465a25442fd64a1ab33d98362d66d25b.html?q=extend%2520service.
+
+## Configuring Quick Buy for Google Pay
+
+To complete the configuration of Quick Buy for Google Pay, you need to define the Google Pay API URL, as described in the following section. You can also modify the credit card parameters for Google Pay, as described below.
+
+### Defining the Google Pay API URL
+
+You need to define the Google Pay API URL, in one of the following ways:
+
+- When installing the open payment framework library with schematics, you can use the `--opfGooglePayApiUrl` parameter, as shown in the following example:
+   ```text
+   --opfGooglePayApiUrl=https://pay.google.com/gp/p/js/pay.js
+   ```
+
+- Alternatively, after installing the open framework library, you can replace the following placeholder in `opf-feature.module.ts`:
+
+```ts
+provideConfig(<OpfQuickBuyConfig>{
+providers:{
+  googlePay: {
+    resourceUrl: "PLACEHOLDER_GOOGLE_PAY_API_URL" // replace the placeholder here
+}
+}
+}),
+```
+
+### Modifying Credit Card Parameters for Google Pay
+
+The card parameters configuration for Google Pay is hard-coded in the `OpfGooglePayService` class in `google-pay.service.ts`. You can modify the configuration by extending `OpfGooglePayService` and overwriting the following object:
+
+```ts
+  protected readonly defaultGooglePayCardParameters: any = {
+    allowedAuthMethods: ['PAN_ONLY', 'CRYPTOGRAM_3DS'],
+    allowedCardNetworks: [
+      'AMEX',
+      'DISCOVER',
+      'INTERAC',
+      'JCB',
+      'MASTERCARD',
+      'VISA',
+    ],
+    billingAddressRequired: true,
+    billingAddressParameters: {
+      format: 'FULL',
+    },
+  };
+```
+
+For more information on extending services in Spartacus, see https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/aa417173fe4a4ba5a473c93eb730a417/465a25442fd64a1ab33d98362d66d25b.html?q=extend%2520service.
```

Comment:
Same as comment above about link to SAP Commerce Cloud docs. This one needs replacing as well, if the one above is replaced.

Reply:
We can use: https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/864a3158bf9f49c99e6196e4e0d27323.html?locale=en-US&q=extend+composable#customizing-services

Reply:
Again, thank you!! 🙂